start.sh

```diff
@@ -86,7 +86,7 @@ if [ "${IS_CRON_POD:-"false",,}" == "false" ]; then
         echo Starting Runserver for development
         export PYTHONPATH="/code:$PYTHONPATH"
         export DJANGO_SETTINGS_MODULE=dashboard.settings
-        exec django-admin runserver --ptvsd 0.0.0.0:${GUNICORN_PORT}
+        exec django-admin runserver --nostatic --ptvsd 0.0.0.0:${GUNICORN_PORT}
```

Comment:
I couldn't find `--nostatic` in the documentation.  What does this option do?  Can you give me a link to the documentation for it?

Reply:
#999 is going to remove this runserver. Maybe that will help out for the issue of the dev instance.

Reply:
This has been removed